data/records/lhcb-ntuples-hackathon.json

```diff
@@ -0,0 +1,1022 @@
+      {
+        "checksum": "adler32:d89a5a62",
+        "size": 799202996,
+        "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/upload/opendata-lhcb-ntupling-service/analysis-productions/merge-requests/1790/outputs/real-production/00243791_00000008_1.dvntuple.root"
+      },
+      {
+        "checksum": "adler32:c6ae5b61",
+        "size": 736021535,
+        "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/upload/opendata-lhcb-ntupling-service/analysis-productions/merge-requests/1790/outputs/real-production/00243791_00000009_1.dvntuple.root"
+      },
+      {
+        "checksum": "adler32:50272739",
+        "size": 631150412,
+        "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/upload/opendata-lhcb-ntupling-service/analysis-productions/merge-requests/1790/outputs/real-production/00243791_00000010_1.dvntuple.root"
+      },
+      {
+        "checksum": "adler32:36f24b55",
+        "size": 477562333,
+        "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/upload/opendata-lhcb-ntupling-service/analysis-productions/merge-requests/1790/outputs/real-production/00243791_00000011_1.dvntuple.root"
+      },
+      {
+        "checksum": "adler32:47c37bd6",
+        "size": 36191935,
+        "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/upload/opendata-lhcb-ntupling-service/analysis-productions/merge-requests/1790/outputs/real-production/00243791_00000012_1.dvntuple.root"
+      },
+      {
+        "checksum": "adler32:640aeb60",
+        "size": 3781425,
+        "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/upload/opendata-lhcb-ntupling-service/analysis-productions/merge-requests/1790/outputs/real-production/00243791_00000013_1.dvntuple.root"
+      },
+      {
+        "checksum": "adler32:090f4cd5",
+        "size": 103037794,
+        "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/upload/opendata-lhcb-ntupling-service/analysis-productions/merge-requests/1790/outputs/real-production/00243791_00000014_1.dvntuple.root"
+      },
+      {
+        "checksum": "adler32:02d5a65b",
+        "size": 253152338,
+        "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/upload/opendata-lhcb-ntupling-service/analysis-productions/merge-requests/1790/outputs/real-production/00243791_00000015_1.dvntuple.root"
+      }
+    ],
+    "magnet_polarity": "MagDown",
+    "methodology": {
+      "description": "<p>These ntulpes were produced by means of the LHCb Ntupling Service. Please see the input configuration files <a href=\"/record/939000\">here</a>."
+    },
+    "publisher": "CERN Open Data portal",
+    "recid": "93901",
+    "relations": [
+      {
+        "recid": "93900",
+        "type": "isChildOf"
+      }
+    ],
+    "run_period": ["2011"],
+    "stripping": {
+      "line": "StrippingLb2LcPiLc2PKPiBeauty2CharmLine",
+      "stream": "BHADRON",
+      "version": "stripping21r1"
+    },
+    "title": "{Lambda_b0}[Lambda_b0 -> {Lambda_cplus}(Lambda_c+ -> {pplus}p+ {Kminus}K-{piplus}pi+) {piminus}pi-]CC",
```

Comment:
For here and other occurrences. Right now this similar decay descriptor is repeated for 4 different "related datasets" (we have 8 related datasets -- 2 decays x 2 years x 2 magnet polarities). I have added the year and magnet polarity to the title, which can be reliably extracted from the `info.yaml` file (and will need to be for other fields in the json file, though I am not sure how those are currently used). I have also made the decay descriptor more readable by removing the Ntuple branch names, which could be implemented algorithmically by removing substrings enclosed in "{}".

Reply:
The title look good :+1: 

Shall we also amend the umbrella record title?  Currently it says:

```
LHCb Ntuples from user request 142
```

Perhaps it should similarly say:

```
[Lambda_b0 -> (Lambda_c+ -> p+ K- pi+) K-]CC ntuples
```


or including request ID:

```
[Lambda_b0 -> (Lambda_c+ -> p+ K- pi+) K-]CC ntuples from user request 142 
```

Reply:
FYI went for the following titles in the PR update. (Since the request number 142 is already prominently displayed in the record, we probably don't need it in the title?)

![image](https://github.com/user-attachments/assets/24bf8faa-e658-4597-96c4-6cf88717955a)

Reply:
I have updated the records on QA so you can check how LaTeX looks:

https://opendata-qa.cern.ch/record/93900

Reply:
Everything looks good from my end now, thanks again for setting this up!